sdrgui/CMakeLists.txt

```diff
@@ -49,7 +49,7 @@ set(sdrgui_SOURCES
     gui/valuedialz.cpp
 
     dsp/scopevis.cpp
-#   dsp/scopevismulti.cpp
+    dsp/scopevismulti.cpp
```

Comment:
Why? This is of no use at this moment.

Reply:
Error compile msvc.

Reply:
Error at the stage of linker, there is a link from other files.